src/test/resources/features/sol/SolDebtDetailsRequest.feature

```diff
@@ -0,0 +1,114 @@
+      | "duty02" |
+    When a debt statement of liability is requested
+
+    Then service returns debt statement of liability data
+      | totalAmountIntDebt | combinedDailyAccrual |
+      | 2156               | 14                   |
+
+    And the 1st sol debt summary will contain
+      | uniqueItemReference | mainTrans | description  | periodEnd  | interestDueDebtTotal | interestRequestedTo | combinedDailyAccrual |
+      | debt003             | 5330      | UI: ChB Debt | 2020-01-01 | 2156                 | 2021-05-05          | 14                   |
+
+    And the 1st sol debt summary will contain duties
+      | debtItemChargeID | subTrans | description            | unpaidAmountDebt | combinedDailyAccrual |
+      | duty01           | 7006     | UI: Child Benefit Debt | 400000           | 0                    |
+      | duty02           | 1000     | IT                     | 200000           | 14                   |
+
+  Scenario: 3. CO: Child Benefit Migrated Debt statement of liability, 1 duty, no payment history.
+    Given debt details
+      | solType | debtId  | mainTrans | subTrans |
+      | CO      | debt004 | 5350      | 7012     |
+    And add debt item chargeIDs to the debt
+      | dutyId   |
+      | "duty04" |
+    When a debt statement of liability is requested
+
+    Then service returns debt statement of liability data
+      | totalAmountIntDebt | combinedDailyAccrual |
+      | 0                  | 0                    |
+
+    And the 1st sol debt summary will contain
+      | uniqueItemReference | mainTrans | description           | periodEnd  | interestDueDebtTotal | interestRequestedTo | combinedDailyAccrual |
+      | debt004             | 5350      | CO: ChB Migrated Debt | 2020-01-01 | 0                    | 2021-05-05          | 0                    |
+
+    And the 1st sol debt summary will contain duties
+      | debtItemChargeID | subTrans | description                     | unpaidAmountDebt | combinedDailyAccrual |
+      | duty04           | 7012     | CO: Child Benefit Migrated Debt | 200000           | 0                    |
+
+
+  Scenario: 4. MainTrans and subTrans non interest bearing - IFS still calculates interest and it is zero
+    Given debt details
+      | solType | debtId  | mainTrans | subTrans |
+      | CO      | debt005 | 5350      | 7012     |
+    And add debt item chargeIDs to the debt
+      | dutyId   |
+      | "duty06" |
+    When a debt statement of liability is requested
+
+    Then service returns debt statement of liability data
+      | totalAmountIntDebt | combinedDailyAccrual |
+      | 0                  | 0                    |
+
+    And the 1st sol debt summary will contain
+      | uniqueItemReference | mainTrans | description           | periodEnd  | interestDueDebtTotal | interestRequestedTo | combinedDailyAccrual |
+      | debt005             | 5350      | CO: ChB Migrated Debt | 2021-07-31 | 0                    | 2021-07-31          | 0                    |
```

Comment:
period end fails test with incorrect date. As per Helen's comment " the matrix shows which values are provided and returned, period end is provided by Pega by not returned by SoL, hope that makes sense"